sensor-core/build.gradle

```diff
@@ -77,13 +77,16 @@ dependencies {
     implementation "androidx.cardview:cardview:1.0.0"
 
     implementation "androidx.annotation:annotation:1.1.0"
-    implementation "androidx.core:core:1.5.0-alpha04"
+    implementation "androidx.core:core-ktx:1.5.0-alpha05"
     implementation "com.google.android.material:material:1.3.0-alpha03"
     implementation "androidx.appcompat:appcompat:1.2.0"
 
     implementation 'com.github.PhilJay:MPAndroidChart:v3.1.0-alpha'
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation "com.google.android.gms:play-services-location:$play_services_location"
 
+    //activity recognition.
+    implementation 'com.google.android.gms:play-services-location:17.1.0'
+
```

Comment:
Why does the sensor code have this many dependencies on UI libraries? Please remove as many dependencies as possible.

Reply:
removed